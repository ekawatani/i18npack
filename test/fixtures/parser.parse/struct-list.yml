- !struct
  _schema: book.json
  title: Alice's Adventures in Wonderland
  auothor: Lewis Carroll

- !struct
  _schema: book.json
  title: The Adventures of Tom Sawyer
  auothor: Mark Twain
