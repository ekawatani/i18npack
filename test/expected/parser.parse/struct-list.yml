- title: Alice's Adventures in Wonderland
  auothor: Lewis Carroll
- title: The Adventures of Tom Sawyer
  auothor: Mark Twain
